src/main/kotlin/com/autonomousapps/internal/utils/moshi.kt

```diff
@@ -189,6 +189,12 @@ inline fun <reified T> File.bufferWriteJson(set: T, indent: String = noJsonInden
   }
 }
 
+inline fun <reified T> File.bufferWriteJson(jsonAdapter: JsonAdapter<T>, data: T, indent: String = noJsonIndent) {
```

Comment:
I'm not sure how I feel about passing in the JsonAdapter here, as it is inconsistent with the other functions in this file. Is it necessary?

Reply:
I've just pushed a second commit to make this consistent. The new method is for any generic type with one type parameter. The signature is a bit strange (with the type param only being used in the function body), but it's not too bad.